src/qt/rpcconsole.cpp

```diff
@@ -88,24 +88,6 @@ const QStringList historyFilter = QStringList()
 
 }
 
-/* Object for executing console RPC commands in a separate thread.
```

Comment:
5aa5ed9897cfe3d5832daa9b25835b73da37f5a9

A forward declaration would be fine for the `m_executor` member (so this commit could be dropped)?

Reply:
Thanks! [Updated](https://github.com/bitcoin-core/gui/pull/589#issuecomment-1105104759).